### DISCUSSION TITLE
<Commit history is not shown for several days>

### DISCUSSION THREAD

**Discussion by devkayes** [community/community#44680] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

On my profile **Graph Contributions** section from last days commit history is not updated at all. All though I check everything (correct email, correct repo) and all are ok. But from few days still  **Commit Graph** isn't update yet.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @its-kayes ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by radar-blip** [comment]
If those commits were made in a private repository, they might not show up (depends on your settings).

**Reply by devkayes** [reply]
> If those commits were made in a private repository, they might not show up (depends on your settings).

@swimshahriar Sir, Please look into this issue.
